### ISSUE TITLE
Can not add games

### ISSUE THREAD

**Issue by jackmiller85** [MichaelHillcox/Echelon#56]
After installed echelon , login as admin and go to game setting , after added new game on game-settings page , all is corectly added . But nothing happen , dont have game just added , and go to page banlist got error 500 .

**Comment by MichaelHillcox** [comment]
Hi, Currently this version of Echelon is in an very unstable state and isn't ready to be running on a production server

**Comment by jackmiller85** [comment]
its just test install , but for add game after finish do not show ID :  http://prntscr.com/kebwuz

**Comment by MichaelHillcox** [comment]
Haha, I know of this one :P If you check your database and look at games, you'll see the id that you need to use

**Comment by jackmiller85** [comment]
yes i did it too before come here , but the problem is there is nothing on this table : http://prntscr.com/kebyeq

**Comment by MichaelHillcox** [comment]
Oh, cool, it completely failed. You can manually insert into that table, I'll keep this as a bug report and see what I can do to fix this issue 👍

**Comment by jackmiller85** [comment]
How to insert manual ? can you give me 1 example ?

**Comment by MichaelHillcox** [comment]
Best thing you can do is wait until I fix it offically :P

**Comment by jackmiller85** [comment]
ok , then stop test ^^ thanks for your time
